### PR TITLE
Changed K+ to K?

### PR DESCRIPTION
**Pull request by Subtlesnow**
Associated Issue: #4213

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

- [x] Took out SVG component that was for K+ and added K? in text because we couldn't find another SVG icon.
- [x] Moved the button to the right.
- [x] Soften Color to match accordion

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ class App extends Component {
   renderVerticalLayout: Function;
   toggleSymbolModal: Function;
   onEscape: Function;
+  onCommandSlash: Function;
```
Comment (codehag) [inline src/components/App.js]: Someone mentioned that there is another command for opening the shortcuts modal... perhaps we should double check before doing this?  @jasonLaster thoughts? maybe this would be better in a separate pr?

File: src/components/SecondaryPanes/CommandBar.css [hunk 2]
```diff
@@ -2,6 +2,7 @@
   flex: 0 0 29px;
   border-bottom: 1px solid var(--theme-splitter-color);
   display: flex;
+  justify-content: flex-end;
```
Comment (codehag) [inline src/components/SecondaryPanes/CommandBar.css]: this also moves all of the stepping buttons over to the right hand side of the bar, which is not something we want to do right now. lets remove this rule

File: src/components/SecondaryPanes/UtilsBar.js [hunk 3]
```diff
@@ -15,7 +14,7 @@ function debugBtn(onClick, type, className, tooltip, disabled = false) {
 
   return (
     <button className={classnames(type, className)} {...props}>
-      <Svg name={type} />
+      K?
```
Comment (codehag) [inline src/components/SecondaryPanes/UtilsBar.js]: I was thinking that the button could be a simple `?` (since im used to it from other editors) with no K (since K is not necessarily meaningful).. not sure, what do you think?

**Comment by jasonLaster** [comment]
Hey @Subtlesnow do you think you'll have time to make these changes?

**Comment by Subtlesnow** [comment]
Yes I should be able to fix them in the next week.

**Comment by jasonLaster** [comment]
Thanks @Subtlesnow!

**Comment by jasonLaster** [comment]
Hey @Subtlesnow how's your availability?

**Comment by Subtlesnow** [comment]
Hello,

I apologize for getting around to fixing those bugs. I have been very busy
this past week.

**Comment by jasonLaster** [comment]
Hey @Subtlesnow, I re-opened your PR on #4382 w/ some small tweaks. Thanks for the help at the meetup!
